hugegraph-core/src/main/java/com/baidu/hugegraph/job/algorithm/SubgraphStatAlgorithm.java

```diff
@@ -95,7 +96,7 @@ private HugeGraph createTempGraph(UserJob<Object> job) {
          * NOTE: this temp graph don't need to init backend because no task info
          * required, also not set started because no task to be scheduled.
          */
-        return new StandardHugeGraph(new HugeConfig(config));
+        return new StandardHugeGraph(new HugeConfig((Configuration) config));
```

Comment:
here the `Configuration` dependency has changed
old: `org.apache.commons.configuration` --> new:"org.apache.commons.configuration2.PropertiesConfiguration;"